src/raftstore/store/worker/compact.rs

```diff
@@ -158,6 +158,8 @@ impl Runnable<Task> for Runner {
                                 "compact range ({:?}, {:?}) for cf {:?} failed, error {:?}",
                                 start, end, cf, e
                             );
+                        } else {
+                            info!("compact range ({:?}, {:?}) for cf {:?} finished", start, end, cf);
```

Comment:
Will it print a lot of information during compaction?

Reply:
It equals to the times of manual compaction in RocksDB. In my observation, the common case is  a few hundreds or thousands per day.

Reply:
do you need to record the compact time here?

Reply:
How about moving the info log inside `compact_range_cf`, there is a timer inside that too, and we can also remove the info log in L136.

Reply:
great idea

/cc @lerencao